core-bundle/assets/scripts/modulewizard.js

```diff
@@ -10,8 +10,9 @@
             return;
         }
 
-        // Check if this row has all necessary elements
-        if (9 !== row.querySelectorAll('select, button, a.module_link, img.module_image').length) {
+        // Check if this row has all necessary elements.
+        // This is to prevent the mutation observer from initializing the incomplete widget.
+        if (row.querySelectorAll('select, a.module_link, img.module_image, button[data-command="copy"], button[data-command="delete"], button.drag-handle').length < 9) {
```

Comment:
I tested this version, but it did not work. `.length` was always `8` in my test.

As this check is about the mutation observer, I think we only need to check for the last element.

```suggestion
        // Check if this row has all necessary elements.
        // This is to prevent the mutation observer from initializing the incomplete widget.
        if (!row.querySelector('button.drag-handle')) {
```